app/src/main/java/com/maq/xprize/onecourse/hindi/mainui/SplashScreenActivity.java

```diff
@@ -33,33 +31,27 @@
     String expansionFilePath;
     File expansionFile;
     ZipFile expansionZipFile;
-    Zip _zip;
+    Zip zipHandler;
     String unzipFilePath;
     File packageNameDir;
     SharedPreferences sharedPref;
     int defaultFileVersion = 0;
-    int mainFileVersion;
-    int patchFileVersion;
+    int storedMainFileVersion;
+    int storedPatchFileVersion;
     boolean isExtractionRequired = false;
 
-    public static String getUnzippedExpansionFilePath() {
-        return "/storage/emulated/0/Android/data/com.maq.xprize.onecourse.hindi/files/";
+    public String getUnzippedExpansionFilePath() {
+        return "/storage/emulated/0/Android/data/" + getPackageName() + "/files/";
```

Comment:
Why not use `getExternalStorageDirectory()` instead of `/storage/emulated/0/`

https://developer.android.com/google/play/expansion-files#StorageLocation